### ISSUE TITLE
Punctuation checkbox in voices dialogue is global to all synths

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#928]
**Reported by briang1 on 2010-09-22 14:56**
Whatever the setting of the punctuation , ie speak all, or not, affects all synthescisers, in other words it is global, whereas the other settings in the voices dialogue are local to the synth selected.

This will perhaps need a rethink when property sheets are redesigned or made more logical later on.

Also of course if the tickets regarding changes to  the configuration of what is and is not spoken and in what context are acted on, this may become academic.

**Comment by leonardder** [comment]
cc @JCSTeh @Brian1Gaff

**Comment by jcsteh** [comment]
We should probably look at reorganising some of those global options once #577 is done and we can have cleaner categorisation of our settings.

**Comment by bdorer** [comment]
Now #577 has been done so I think this could be done easily.

**Comment by Adriani90** [comment]
cc: @feerrenrut

**Comment by feerrenrut** [comment]
It's not really clear what this issue is about. I would like to see a better description of the problem to be solved here. Please tell a story that explains a situation where the punctuation level should be tied to the synthesizer rather than a global setting. 

I think @jcsteh was suggesting that this setting should remain global, but be moved to another category in order to clarify that it is not synth specific. If anyone wishes to elaborate on this, please do so on another issue.

**Comment by Adriani90** [comment]
My understanding is that this issue is more about consistency and not about use cases. I agree with Jamie that the punctuation could be moved to another cathegory, maybe entirely out of the settings to the voice dictionary or to the symbol pronounciation.

**Comment by feerrenrut** [comment]
In that case I think we should close this issue and have a more targeted discussion elsewhere. At the moment this issue doesn't have a clear goal.
